diffsims/crystallography/reciprocal_lattice_vector.py

```diff
@@ -502,6 +503,11 @@ def scattering_parameter(self):
 
         return 0.5 * self.gspacing
 
+    def rotate_from_matrix(self, rotation_matrix):
+        return ReciprocalLatticeVector(
+            phase=self.phase, xyz=np.matmul(rotation_matrix.T, self.data.T).T
```

Comment:
This should be `np.matmul(self.data, rotation_matrix.T)`, no?

That is in line with orix. Compare e.g.

```python
import numpy as np
from orix.crystal_map import Phase
from orix.quaternion import Rotation
from diffsims.crystallography import ReciprocalLatticeVector


phase = Phase(point_group="m-3m")
g = ReciprocalLatticeVector.from_highest_hkl(phase, [1, 1, 1])
g_m = g.to_miller()

R = Rotation.random()
om = R.to_matrix().squeeze()

g1 = R * g_m
g2 = ReciprocalLatticeVector(
    phase=g.phase, xyz=np.matmul(g.data, om.T)
)

np.allclose(g1.data, g2.data)
# True
```

Reply:
What would the difference be?  I think this would change the simulation results again... @viljarjf do you have a better idea of what this might change?

Reply:
I added a test to check that the results remain similar. Maybe @viljarjf you have a comment on this

Reply:
I am a fan of staying consistent with orix, at least. Preferably, the ReciprocalLatticeVector should be rotated by multiplying with a Rotation, rather than a rotation matrix, but this currently does not work as reported in #211.
Transposing the rotation matrix inverts the rotation, which might be why the simulations change.
As for the test you added, I believe the .npy-file is missing right now?

Reply:
@viljarjf i think the file is there but it's being excluded in the build? @hakonanes Might have a better idea